src/Taxjar/Taxjar.csproj

```diff
@@ -22,7 +22,6 @@
 
   <ItemGroup>
     <Folder Include="Entities\" />
-    <Folder Include="Infrastructure\" />
```

Comment:
@codymullins Is this folder removal from the project needed?

Reply:
Nope, I didn't even remove this, nice catch. VS must have auto removed for some reason?